hosted-ce/30-remote-site-setup.sh

```diff
@@ -111,12 +123,12 @@ REMOTE_HOST_KEY=`ssh-keyscan -p "$remote_port" "$remote_fqdn"`
 root_ssh_dir=/root/.ssh/
 mkdir -p $root_ssh_dir
 chmod 700 $root_ssh_dir
-ln -s $BOSCO_KEY $root_ssh_dir/bosco_key.rsa
+ln -s "$(get_bosco_key "root")" $root_ssh_dir/bosco_key.rsa
 
 cat <<EOF > /etc/ssh/ssh_config
 Host $remote_fqdn
   Port $remote_port
-  IdentityFile ${BOSCO_KEY}
+  IdentityFile "$(get_bosco_key "root")"
```

Comment:
Are users going to have to create their secret with a `root` user and corresponding key?

I think we may be able to drop this rootly SSH config as long as we:

1. Run remote OS detection using one of the other users
2. Set up the hacky `bosco_key.rsa` per user
3. Run `bosco_cluster` and env file `rsync` as the unpriv users

Reply:
I am totally open to using one of the other users instead of root for this.
Doesn't bosco_cluster have to be run once per user anyway?

Reply:
Yes but we currently run it as root, not as the unpriv user https://github.com/opensciencegrid/docker-compute-entrypoint/blob/d78931ebbd7bdfe6861f0ecc7534df034d1e30c3/hosted-ce/30-remote-site-setup.sh#L198-L202